pi-memory/tests/test_recall_search.py

```
@@ -0,0 +1,236 @@
+    )
+
+
+def add_indexed_transcript(
+    database: Database,
+    *,
+    session_id: str,
+    path: str,
+    entries: list[TranscriptEntry],
+) -> tuple[int, list[int]]:
+    with database.session() as session:
+        memory_session = MemorySession(session_id=session_id, cwd="/tmp/project", repo_name="basecamp")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/project"

(S108)